stories/examples/controlled.js

```diff
@@ -0,0 +1,207 @@
+import React, {Component} from 'react'
+import glamorous, {Div} from 'glamorous'
+import matchSorter from 'match-sorter'
+import Autocomplete from '../../src'
+
+const Input = glamorous.input({
+  fontSize: 14,
+  wordWrap: 'break-word',
+  lineHeight: '1em',
+  outline: 0,
+  whiteSpace: 'normal',
+  minHeight: '2em',
+  background: '#fff',
+  display: 'inline-block',
+  padding: '.5em 2em .5em 1em',
+  color: 'rgba(0,0,0,.87)',
+  boxShadow: 'none',
+  border: '1px solid rgba(34,36,38,.15)',
+  transition: 'box-shadow .1s ease,width .1s ease',
+  margin: 0,
+  marginBottom: '-2px',
+  '&:hover, &focus': {
+    borderColor: 'rgba(34,36,38,.35)',
+    boxShadow: 'none',
+  },
+})
+
+class Examples extends Component {
+  state = {
+    selectedColor: '',
+    inputValue: '',
+    isOpen: false,
+  }
+  items = ['Black', 'Red', 'Green', 'Blue', 'Orange', 'Purple']
+
+  changeHandler = changes => {
+    let {
+      selectedValue = this.state.selectedColor,
+      isOpen = this.state.isOpen,
+      inputValue = this.state.inputValue,
+      type,
+    } = changes
+    isOpen =
```

Comment:
Here's where the `type` comes in handy. This allows us to know whether the open/close happened due to a mouseup interaction, in which case we want to leave it open because we're controlling that state ourselves. Otherwise you'd click the button and that would count as a mouseup.